powerstore/resource_snapshot_test.go

```diff
@@ -40,8 +42,7 @@ func TestAccVolumeSnapshot_CreateWithoutName(t *testing.T) {
 		ProtoV6ProviderFactories: testProviderFactory,
 		Steps: []resource.TestStep{
 			{
-				Config:      ProviderConfigForTesting + SnapshotParamsCreateWithoutName,
-				ExpectError: regexp.MustCompile(CreateResourceMissingErrorMsg),
+				Config: ProviderConfigForTesting + SnapshotParamsCreateWithoutName,
```

Comment:
should we not add check to validate attributes, if this is passing case. applicable to "TestAccVolumeSnapshot_CreateWithoutExpiration" as well.